### ISSUE TITLE
GCP CLI Orb 1.5.0 failure

### ISSUE THREAD

**Issue by clakech** [CircleCI-Public/gcp-cli-orb#9]
### Orb version

1.5.0

### What happened

Cannot install lsb-release so CLOUD_SDK_REPO = "cloud-sdk-" so gcp cli install fails

When the orb run "$SUDO apt-get update && $SUDO apt-get -y install lsb-release"
Here is the error: "E: Could not open file /var/lib/apt/lists/deb.debian.org_debian_dists_stretch-backports_main_binary-amd64_Packages.diff_Index - open (2: No such file or directory)"

You should fix apt to be able to install lsb-release properly

Here are the logs:

```
#!/bin/bash -eo pipefail
install () {
  # Set sudo to work whether logged in as root user or non-root user
  if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi

  # Create an environment variable for the correct distribution
  if [[ $(command -v lsb_release) == "" ]]; then
    $SUDO apt-get update && $SUDO apt-get -y install lsb-release
    export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)"
  else
    export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)"
  fi

  # Add the Google Cloud SDK distribution URI as a package source
  echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | $SUDO tee -a /etc/apt/sources.list.d/google-cloud-sdk.list

  # Import the Google Cloud public key
  if [[ $(command -v curl) == "" ]]; then
    $SUDO apt-get update && $SUDO apt-get -y install curl
    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | $SUDO apt-key add -
  else
    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | $SUDO apt-key add -
  fi

  # Update and install the Cloud SDK
  $SUDO apt-get update && $SUDO apt-get -y install google-cloud-sdk
}

if grep 'docker\|lxc' /proc/1/cgroup > /dev/null 2>&1; then
  if [[ $(command -v gcloud) == "" ]]; then
    install
  else
    echo "gcloud CLI is already installed."
  fi
else
  echo "----------------------------------------------------------------------------------------------------"
  echo "this is a machine executor job, replacing default installation of gcloud CLI"
  echo "----------------------------------------------------------------------------------------------------"
  sudo rm -rf /opt/google-cloud-sdk
  install
fi

echo "Success! gcloud has been installed; displaying version information:"
gcloud version
Get:1 http://security.debian.org/debian-security stretch/updates InRelease [94.3 kB]

Ign:2 http://deb.debian.org/debian stretch InRelease

Get:3 http://deb.debian.org/debian stretch-updates InRelease [91.0 kB]

Get:4 http://deb.debian.org/debian stretch-backports InRelease [91.8 kB]

Get:5 http://deb.debian.org/debian stretch Release [118 kB]

Get:6 http://deb.debian.org/debian stretch Release.gpg [2434 B]

Get:7 http://deb.debian.org/debian stretch-updates/main amd64 Packages.diff/Index [10.1 kB]

Get:8 http://deb.debian.org/debian stretch-updates/main amd64 Packages [27.2 kB]

Get:9 http://deb.debian.org/debian stretch-backports/main amd64 Packages.diff/Index [27.8 kB]

Get:9 http://deb.debian.org/debian stretch-backports/main amd64 Packages.diff/Index [27.8 kB]

Get:10 http://deb.debian.org/debian stretch-backports/main amd64 Packages [601 kB]

Get:11 http://security.debian.org/debian-security stretch/updates/main amd64 Packages [499 kB]

Get:12 http://deb.debian.org/debian stretch/main amd64 Packages [7082 kB]

Reading package lists... Done

E: Could not open file /var/lib/apt/lists/deb.debian.org_debian_dists_stretch-backports_main_binary-amd64_Packages.diff_Index - open (2: No such file or directory)
environment: line 7: lsb_release: command not found
deb http://packages.cloud.google.com/apt cloud-sdk- main
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

100  1326  100  1326    0     0  24888      0 --:--:-- --:--:-- --:--:-- 25018
OK

Ign:1 http://deb.debian.org/debian stretch InRelease

Hit:2 http://deb.debian.org/debian stretch-updates InRelease

Hit:3 http://deb.debian.org/debian stretch-backports InRelease

Hit:4 http://deb.debian.org/debian stretch Release

Ign:5 http://packages.cloud.google.com/apt cloud-sdk- InRelease

Ign:6 http://packages.cloud.google.com/apt cloud-sdk- Release

Ign:7 http://packages.cloud.google.com/apt cloud-sdk-/main all Packages

Ign:8 http://packages.cloud.google.com/apt cloud-sdk-/main amd64 Packages

Ign:7 http://packages.cloud.google.com/apt cloud-sdk-/main all Packages

Ign:8 http://packages.cloud.google.com/apt cloud-sdk-/main amd64 Packages

Ign:7 http://packages.cloud.google.com/apt cloud-sdk-/main all Packages

Ign:8 http://packages.cloud.google.com/apt cloud-sdk-/main amd64 Packages

Hit:9 http://security.debian.org/debian-security stretch/updates InRelease

Ign:7 http://packages.cloud.google.com/apt cloud-sdk-/main all Packages

Ign:8 http://packages.cloud.google.com/apt cloud-sdk-/main amd64 Packages

Ign:7 http://packages.cloud.google.com/apt cloud-sdk-/main all Packages

Ign:8 http://packages.cloud.google.com/apt cloud-sdk-/main amd64 Packages

Ign:7 http://packages.cloud.google.com/apt cloud-sdk-/main all Packages

Err:8 http://packages.cloud.google.com/apt cloud-sdk-/main amd64 Packages
  404  Not Found

Reading package lists... Done

W: The repository 'http://packages.cloud.google.com/apt cloud-sdk- Release' does not have a Release file.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: Failed to fetch http://packages.cloud.google.com/apt/dists/cloud-sdk-/main/binary-amd64/Packages  404  Not Found
E: Some index files failed to download. They have been ignored, or old ones used instead.
Exited with code 100
```

### Expected behavior

should work as usual ;)

**Comment by iynere** [comment]
@clakech what type of execution environment were you using when you saw this error ? i'm guessing you're using a docker image—which docker image ? thank you!

**Comment by clakech** [comment]
We are using this docker image circleci/openjdk:11-jdk

**Comment by iynere** [comment]
patched in `circleci/gcp-cli@1.8.2`
